product_docs/docs/edbcloud/beta/reference/index.mdx

```diff
@@ -0,0 +1,193 @@
+
+- `refresh_token` - use to obtain a new access token or ID token after the previous one has expired. (See [Refresh Tokens](https://auth0.com/docs/tokens/refresh-tokens) for more information.) Refresh tokens expire after 30 days.
+!!! Note
+    Store your refresh token securely so you can use it if you need to get a new access token.  
+- `expires_in` - means the token expires after 24 hours since its creation.
+
+For example:
+```
+{
+  "access_token": "eyJhbGc.......1Qtkaw2fyho",
+  "id_token": "eyJhbGci.......FBra7tA",
+  "refresh_token": "v1.MTvuZpu.......sbiionEhtTw",
+  "scope": "openid profile email offline_access",
+  "expires_in": 86400,
+  "token_type": "Bearer"
+}
+```
+
+
+If not successful, you receive one of the following errors:
+- `authorization_pending` - continue polling using the suggested interval retrieved when you [requested the device code](#request-the-device-code).
+
+- `slow_down` - slow down and use the suggested interval retrieved when you [requested the device code](#request-the-device-code). To avoid receiving this error due to network latency, you should start counting each interval after receipt of the last polling request's response.
+- `expired_token` - you have not authorized the device quickly enough, so the `device_code` has expired. Your application should notify you that it has expired and to restore it.
+- `access_denied` 
+
+Contact [Customer Support](../overview/06_support) if you have trouble obtaining a valid access token to access EDB Cloud API. 
+
+## Calling the API
+
+To call the EDB Cloud API, your application must pass the retrieved access token as a bearer token in the Authorization header of your HTTP request. For example: 
+
+```
+curl --request GET \
+  --url https://portal.edbcloud.com/api/v1/postgres-types \
+  --header 'authorization: Bearer <access_token>  \
+  --header 'content-type: application/json'
+```
+
+Example response:
+```
+{
+  "pgTypesList": [
+    {
+      "id": "epas",
+      "name": "EDB Postgres Advanced Server",
+      "description": "",
+      "dockerImage": ""
+    },
+    {
+      "id": "postgres",
+      "name": "PostgreSQL",
+      "description": "",
+      "dockerImage": ""
+    }
+  ]
+}
+```
+
+
+## Refreshing your Token
+
+You can use the refresh token to get a new access token. Usually you need a new access token only after the previous one expires or when gaining access to a new resource for the first time. You shouldn't call the endpoint to get a new access token every time you call an API. There are rate limits that throttle the amount of requests to the endpoint that can be executed using the same token from the same IP. 
+
+To refresh your token, make a POST request to the `/oauth/token` endpoint in the Authentication API, using `grant_type=refresh_token`. For example:
+
+```
+curl --request POST \
+  --url 'https://edbcloud.us.auth0.com/oauth/token' \
```

Comment:
plz change this to: `https://auth.edbcloud.com/oauth/token`, we are going to use customized domain name

Reply:
See latest commit